stories/Atom/Global/Color/color.stories.mdx

```diff
@@ -0,0 +1,281 @@
+    case 'japanese': const japaneseText = {PC: '原色', AC: 'アクセントカラー', BC: 'カラーパレットのバランス', PT: '主要な'}; return japaneseText;
+    default:
+      return {PC: 'Primary colors', AC: 'Accent colors', BC: 'Balance of color palette', PT: 'Primary'};
+  }
+};
+
+<Meta title="Atoms/Colors" component={Color} />
+
+<style>{`
+  div#story--atoms-colors--colors {
+    display: none;
+}
+  .subheading {
+    font-weight: 700;
+    font-size: 18px;
+    color: #666;
+    letter-spacing: 1.5px;
+    line-height: 24px;
+    margin-bottom: 12px;
+    margin-top: 40px;
+  } 
+  .color_wrapper{
+    display: flex;
+  }
+  .color_item{
+    margin: 5px;
+  }
+  .color_box {
+    width: 90px;
+    height: 60px;
+    display: block;
+  }
+  .color_name{
+   font-size: 13px;
+   color: #666;
+   font-weight: 600;
+  margin: 10px 0;
+  }
+  .color_code{
+    font-size: 13px;
+    color: #666;
+  }
+
+  .p-blue-100 { background-color: #B5D5F5;}
+  .p-blue-200 {background-color: #94C4F5;}
+  .p-blue-300 { background-color: #6BABEB;}
+  .p-blue-400 {background-color: #4F95DD;}
+  .p-blue-500 {background-color: #3288CE;}
+  .p-blue-600 {background-color: #006EB5;}
+  .p-blue-700 {background-color: #1F5A95;}
+
+  .p-gray-100 { background-color: #FAFAFA;}
+  .p-gray-200 {background-color: #F7F7F7;}
+  .p-gray-300 { background-color: #EDEFF0;}
+  .p-gray-400 {background-color: #D4D6D8;}
+  .p-gray-500 {background-color: #A9B1B7;}
+  .p-gray-600 {background-color: #55606E;}
+  .p-gray-700 {background-color: #232E3D;}
+
+  .light-yellow { background-color: #FFE17E;}
+  .yellow {background-color: #FFEB00;}
+  .dark-yellow  { background-color: #FBC412;}
+  .light-red {background-color: #FFBCB7;}
+  .red {background-color: #EE402D;}
+  .dark-red {background-color: #D12800;}
+  .light-green {background-color: #B8ECB6;}
+  .green {background-color: #6DE354;}
+  .dark-green {background-color: #59BA47;}
+  .light-azure {background-color: #A2DAF3;}
+  .azure {background-color: #60D4F2;}
+  .dark-azure {background-color: #00C1FF;}
+
+`}</style>
+
+# Color Component
+Color component represent the all colors we are used into the website. we used all the colors as variables.
+
+<ul>
+<li>colors are showed in the Primary and Accents colors option, this variables are used in component.</li>
+<li>colors variables are saved in <code>assets/scss/variables</code> folder file in SCSS file format. and its managable by this file all over the site.</li>
+</ul>
+
+<div className="subheading">Primary Colors</div>
+
+Primary colours are the base colours of the UI, the ones that are used most frequently in the interface. it is used for brand elements and non-interactive elements. Interactive colours are used for primary links and action items that can be clicked or touched, to emphasize important actions and highlight information in your application.
+
+<div className="subheading">How to Use </div>
```

Comment:
@harishkumar-srijan can we remove this section from here and add its content in the usage